src/main/java/ecdar/presentations/QueryPresentation.java

```diff
@@ -254,12 +254,19 @@ private void initializeStateIndicator() {
                 if (controller.getQuery().getQuery().isEmpty()) return;
                 
                 Label label = new Label(tooltip.getText());
-                JFXDialog dialog = new InformationDialogPresentation("Result from query: " + controller.getQuery().getQuery(), label);
+                //Make sure that certain symbols in the query are displayed correctly
+                String newString = changeRefinementSymbols();
```

Comment:
and here?

Reply:
det er for at vise det som unicode i tooltippet

Reply:
Kan i ikke bruge changeRefinementSymbols begge steder?

Reply:
jo hvis vi laver den public

Reply:
I kan eventuel lave den public static på Query

Reply:
Så kan i også bruge den her: https://github.com/Ecdar-SW5/Ecdar-GUI/pull/22/files#diff-5646449d70254cf84ee85bddbe2e55400d64189b778cc38b5b4befad884e010aR62